<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Weekly_Stats_Application</class>
 <widget class="QWidget" name="Weekly_Stats_Application">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Weekly_Stats_Application</string>
  </property>
  <property name="toolTipDuration">
   <number>-5</number>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>551</width>
     <height>164</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="Week_Selection">
    <item row="0" column="1">
     <widget class="QDateEdit" name="week_start">
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2020</year>
        <month>4</month>
        <day>13</day>
       </datetime>
      </property>
      <property name="maximumDateTime">
       <datetime>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
        <year>2050</year>
        <month>12</month>
        <day>31</day>
       </datetime>
      </property>
      <property name="maximumDate">
       <date>
        <year>2050</year>
        <month>12</month>
        <day>31</day>
       </date>
      </property>
      <property name="minimumDate">
       <date>
        <year>2018</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2020</year>
        <month>4</month>
        <day>13</day>
       </date>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="ok_btn">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="ok_btn_label">
      <property name="text">
       <string>Press OK to use selected dates</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="week_end_label">
      <property name="text">
       <string>Week End</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateEdit" name="week_end">
      <property name="dateTime">
       <datetime>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
        <year>2020</year>
        <month>4</month>
        <day>13</day>
       </datetime>
      </property>
      <property name="maximumDate">
       <date>
        <year>2050</year>
        <month>12</month>
        <day>31</day>
       </date>
      </property>
      <property name="minimumDate">
       <date>
        <year>2018</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2020</year>
        <month>4</month>
        <day>13</day>
       </date>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="week_start_label">
      <property name="text">
       <string>Week Start</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="get_data_label">
      <property name="text">
       <string>Press this button to get trip data for week</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="get_data_btn">
      <property name="text">
       <string>Get Data</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="Exclusion_Dates_btns">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
